pybandits/offline_policy_evaluator.py

```diff
@@ -0,0 +1,855 @@
+from functools import partial
+from itertools import product
+from math import floor
+from multiprocessing import Pool, cpu_count
+from typing import Any, Dict, List, Literal, Optional, TypeVar, Union
+
+import numpy as np
+import optuna
+import pandas as pd
+from loguru import logger
+from obp.ope import OffPolicyEvaluation
+from obp.ope.estimators import (
```

Comment:
Would it be possible to extend to additional methods as available in obp library: https://zr-obp.readthedocs.io/en/latest/estimators.html
I tested alternative strategies to DM, IPS and DR but encounter compilation errors most of the time. Could you test them and see if you can manage the "errors"?

Reply:
The class now works with all available estimators on OBP package.